src/app/plan/components/Wizard/WizardContainer.tsx

```diff
@@ -66,13 +75,23 @@ export interface IOtherProps {
     persistentVolumes: IPlanPersistentVolume[],
     sourceClusterName: IFormValues['sourceCluster']
   ) => void;
+  fetchHooksRequest: (currentPlanHooks) => void;
   addPlanRequest: (migPlan) => void;
   sourceClusterNamespaces: ISourceClusterNamespace[];
   pvResourceList: IPersistentVolumeResource[];
   onHandleWizardModalClose: () => void;
   editPlanObj?: any;
   isEdit: boolean;
   updateCurrentPlanStatus: any;
+  addHookRequest: (migHook) => void;
+  updateHookRequest: (migHook) => void;
```

Comment:
Ideally each of these callback arguments would have type annotations too, they are implicitly `any` here